### DISCUSSION TITLE
How to use a custom LLM wrapper in Langflow ?

### DISCUSSION THREAD

**Discussion by kitchxia** [langflow-ai/langflow#467] [Q&A]
https://python.langchain.com/en/latest/modules/models/llms/examples/custom_llm.html

I can use my own LLM like this
how can i make it work in Langflow?

**Comment by joelr45** [comment]
Yeah, would love some insight on this, I have tried to create a custom component, but there does not seem to be a way to integrate my own llm.

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
